src/main/resources/drivers/CiscoNext-GenSeries.js

```diff
@@ -0,0 +1,478 @@
+/**
+ * Cisco Catalyst Next-Gen Series (Cisco Business-like CLI) Driver for Netshot
+ * Target: Catalyst 1200/1300 Series (e.g., C1200-16T-2G)
+ *
+ * Author: David Micallef (Pizu)
+ * Version: 1.2 (ES6+)
+ *
+ * Notes:
+ * - Login flow via mini-modes + enable macro (no ssh.targets).
+ * - No VRF: these platforms don’t support it; helper won’t add vrf.
+ * - Robust “System *” capture handles wrapped lines.
+ * - Firmware regex accepts 2.1.1.1, 2.1.1.1.SPA, v2.1.1.1, etc.
+ * - Family derived from System Description or Inventory DESCR, with fallback.
+ */
+
+/* ========================= DRIVER META ========================= */
+
+const Info = {
+  name: "CiscoNextGenSeries",
+  description: "Cisco Catalyst Next-Gen Series",
+  author: "DMica(Pizu)",
+  version: "1.2"
+};
+
+/* ========================= CONFIG MAP ========================= */
+
+const Config = {
+  runningConfig: {
+    type: "LongText",
+    title: "Running configuration",
+    comparable: true,
+    searchable: true,
+    checkable: true,
+    dump: {
+      pre: "!! Running configuration (taken on %when%):",
+      post: "!! End of running configuration"
+    }
+  },
+  firmwareVersion: {
+    type: "Text",
+    title: "Firmware version",
+    comparable: true,
+    searchable: true,
+    checkable: true
+  }
+};
+
+// Keep Device empty to avoid redefining built-in attributes (name, family, etc.)
+const Device = {};
+
+/* ========================= CLI LAYER ========================= */
+
+const PROMPT_EXEC   = /^.*>\s?$/m;
+const PROMPT_ENABLE = /^.*(?:\(config[^)]*\))?#\s?$/m;
+
+const PAGER_RE = new RegExp(
+  "(?:^|\\r?\\n)" +
+  "(?:" +
+    "\\-\\-More\\-\\-" +
+    "|More:\\s*<space>.*" +
+    "|\\[?more\\]?\\s*$" +
+  ")",
+  "mi"
+);
+
+const CLI = {
+  // Transport definition: macro walks options until target mode is reached.
+  ssh: {
+    macros: {
+      enable: {
+        // Order matters: clear banners first, then exec → enable.
+        options: ["username", "password", "anykey", "exec", "enable"],
+        target: "enable"
+      }
+    }
+  },
+
+  /* ---------- Small, single-purpose modes for login banners ---------- */
+
+  username: {
+    // Matches "User Name:", "Username:", "login:"
+    prompt: /(User\s*Name|Username|login):\s*$/mi,
+    response: "%username%"
```

Comment:
Does this really work? Looks like IA-generated code :-) `response` is in `pager` context. There should be macros in each mode to transition to the target one.